src/site/stages/build/process-cms-exports/tests/transformed-entities/node-page.json

```diff
@@ -49,34 +49,49 @@
   "entityPublished": true,
   "entityMetaTags": [
     {
-      "type": "MetaValue",
       "key": "title",
+      "type": "MetaValue",
```

Comment:
There were some new metatags, so I got lazy and copied all of them from the output. This apparently meant these properties are now alphabetized. :man_shrugging: